foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -6,12 +6,13 @@ Let's see how the orientation of items within a flex container can be controlled
 
 This section contains a general overview of topics that you will learn in this lesson.
 
--   You'll learn about the 2 "axes" of a flex container.
--   You'll learn how to change those axes to arrange your content in columns instead of rows.
+- You'll learn about the 2 "axes" of a flex container.
+- You'll learn how to change those axes to arrange your content in columns instead of rows.
 
 The most confusing thing about flexbox is that it can work either horizontally or vertically, and some rules change a bit depending on which direction you are working with.
```

Comment:
@MaoShizhong, instead of omitting this information, I'm thinking of turning it into a note. It provides good advice on what to be aware of when working with Flexbox. Thoughts?

Reply:
Not really sure why those two paragraphs and code block are in the lesson overview and not the `Axes` section...odd. They can definitely be moved down to the start of the `Axes` section. i.e. 

````markdown
### Axes

The most confusing thing about...

The default direction...

```css
/* the css */
```

No matter which direction...
````